### PR TITLE
Rek make advance packet

### PR DESCRIPTION
**Pull request by kilbergr**
## Description

This PR is to merge the shipment summary, uploaded orders, and blank direct deposit form if desired into a PDF known as the Advanced Packet. Currently easiest to test by running via command line. See instructions below.

## Reviewer Notes

To run locally, use a command like the following:
`go run cmd/paperwork/shipment_summary.go -move 00d69a68-5866-475b-ba3b-9eca67f29128`

You must pass in a moveID from your local dev db.

Running in prod will require setting other flags (majority on S3).

## Code Review Verification Steps

* [x] All tests pass.
* [x] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/blob/master/docs/backend.md#logging)
* [x] Request review from a member of a different team.
* [ ] (TRIAL) Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/158568794) for this change

### CODE REVIEW

File: pkg/paperwork/generator.go [hunk 1]
```diff
@@ -170,15 +184,44 @@ func (g *Generator) GenerateAdvancePaperwork(moveID uuid.UUID) (string, error) {
...
+
+	var inputFiles stringSlice
+	g.logger.Debug("adding orders and shipment summary to packet", zap.Any("inputFiles", inputFiles))
+	inputFiles = append(ordersPaths, generatedPath)
+
+	for _, ppm := range move.PersonallyProcuredMoves {
+		if ppm.Advance.MethodOfReceipt == models.MethodOfReceiptOTHERDD {
+			g.logger.Debug("adding direct deposit form to packet", zap.Any("inputFiles", inputFiles))
+			ddFormPath := filepath.Join(build, "/downloads/direct_deposit_form.pdf")
+			inputFiles = append(inputFiles, ddFormPath)
```
Comment (stangah) [inline pkg/paperwork/generator.go]: If you have 5 PPMs, do you want to append the DD form 5 times?

File: pkg/paperwork/generator.go [hunk 2]
```diff
@@ -153,8 +155,20 @@ func (g *Generator) pdfFromImages(images []inputFile) (string, error) {
 	return outputFile.Name(), nil
 }
 
+type stringSlice []string
+
+func (i *stringSlice) String() string {
+	return ""
```
Comment (stangah) [inline pkg/paperwork/generator.go]: What does this method do? Also it's not used.

File: pkg/paperwork/generator.go [hunk 3]
```diff
@@ -153,8 +155,20 @@ func (g *Generator) pdfFromImages(images []inputFile) (string, error) {
...
+}
+
+func (i *stringSlice) Set(value string) error {
+	*i = append(*i, value)
```
Comment (stangah) [inline pkg/paperwork/generator.go]: TIL how `*i` syntax works. Nifty. Doesn't seem like you use this method though.

File: pkg/paperwork/generator.go [hunk 4]
```diff
@@ -153,8 +155,20 @@ func (g *Generator) pdfFromImages(images []inputFile) (string, error) {
 	return outputFile.Name(), nil
 }
 
+type stringSlice []string
```
Comment (stangah) [inline pkg/paperwork/generator.go]: It's unclear what this gives you over using `[]string` directly

File: pkg/paperwork/generator.go [hunk 5]
```diff
@@ -170,15 +184,44 @@ func (g *Generator) GenerateAdvancePaperwork(moveID uuid.UUID) (string, error) {
...
+	}
+
+	mergedFile, err := g.newTempFile()
+	if err != nil {
+		return "", err
+	}
+
+	var inputFiles stringSlice
+	g.logger.Debug("adding orders and shipment summary to packet", zap.Any("inputFiles", inputFiles))
+	inputFiles = append(ordersPaths, generatedPath)
```
Comment (stangah) [inline pkg/paperwork/generator.go]: `ordersPaths` is of type `[]string`, but you instantiated `inputFiles` as a `stringSlice`. I'm not really sure what happens to the type of `inputFiles` after you call `append` on `ordersPaths`, though it doesn't seem like any of the `stringSlice` features are being used so maybe it doesn't matter.

File: pkg/paperwork/generator.go (hunk 1 above)
Comment (kilbergr) [reply pkg/paperwork/generator.go]: No--had a break that I lost. 😬

File: pkg/paperwork/generator.go (hunk 3 above)
Comment (kilbergr) [reply pkg/paperwork/generator.go]: I copied from a stub of Jim's and looks like I got some extra....

**Pull request merged**